### DISCUSSION TITLE
新建以及加载Project没反应，有报错

### DISCUSSION THREAD

**Discussion by AmossXu** [galacean/discussion#106] [Editor]
![image](https://github.com/galacean/discussion/assets/29244256/3788dfe4-dcc3-40cd-89ab-3db1c677798f)

**Comment by AmossXu** [comment]
换了edge可以，我chrome的版本号：116.0.5845.188

**Comment by eyworldwide** [comment]
用隐身模式试试看，可能是浏览器插件影响
